crates/iceberg/src/table.rs

```diff
@@ -63,3 +64,99 @@ impl Table {
         TableScanBuilder::new(self)
     }
 }
+
+/// `StaticTable` is a read-only table struct that can be created from a metadata file or from `TableMetaData` without a catalog.
+/// It can only be used to read metadata and for table scan.
+/// # Examples
+///
+/// ```rust, no_run
+/// # use iceberg::io::FileIO;
+/// # use iceberg::table::StaticTable;
+/// # use iceberg::TableIdent;
+/// # tokio_test::block_on(async {
```

Comment:
We can avoid introducing `tokio_test` by writing an async function:

```rust
#async fn test() {
    let metadata_file_location = "s3://bucket_name/path/to/metadata.json";
    ...
#}
```

Reply:
@Xuanwo - Great idea. Thanks.
Will remove the dependency